### DISCUSSION TITLE
why is vlc useless in armbian and jr ubuntu?

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#1052] [General]
just a general question, not aimed at JR and not an issue, just curious. so vlc has been my no1 player for years, it still is when on windows or mint. but on armbian or ubuntu for opi5 it just doesnt work. or maybe it doesnt work for me. mpv is fine but clunky. how come mpv can work with 2160p 265 files and vlc has trouble with 1080p 265? is that an vlc/arm issue? maybe i might be wrong but i thnk it actualy worked on first versions of armbian for opi5?

**Comment by defencedog** [comment]
Because it doesnt use `mpv`
